app/views/users/show.html.erb

```diff
@@ -1,2 +1,25 @@
-<h1>Users#show</h1>
-<p>Find me in app/views/users/show.html.erb</p>
\ No newline at end of file
+
+<div id='current_user'>
+  <div id='user'>
+    <div class='user_pic'><%= @user.photo%></div>
+    <h2>User <%= @user.name %></h2>
+    <p>Number of posts: <%= @user.posts_count %></p>
+  </div>
+
+  <div>
+    Bio:<p><%= @user.bio %>...</p>
+  </div>
+
+  <ul class='posts'>
+    <% @posts_list.each do |u| %>
+    <div class='post'>
+      <h3>Post # <%= u.id %></h3>
+      <p> Text:<%= u.text %></p>
+      <div>
+        Comments: <%= u.comments_count %>
+        Likes: <%= u.likes_count %>
+      </div>
+    </div>
+    <% end %>
+  </ul>
```

Comment:
**Changes Required**
- [ ] Let us add a **See all Post** links that opens `user's post index` as described in the requirements.